### PR TITLE
Tb/action emphase

### PR DESCRIPTION
**Pull request by tbarthel-fr**
Animations has been added for every users.

In addition, drop figure has been added for kids and used in the ic_launcher as well as empty view indicator.

icons:
device-2016-07-23-192027

for kids : 
device-2016-07-23-192111

default : 
device-2016-07-23-192157

### CODE REVIEW

**Comment by jo-elimu** [comment]
The new icon looks great :-)

File: CameraColorPicker/app/src/kids/res/layout/view_palette_list_page.xml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<merge xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <FrameLayout
+        android:id="@+id/view_palette_list_empty_view"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
```
Comment (vbarthel-fr) [inline CameraColorPicker/app/src/kids/res/layout/view_palette_list_page.xml]: I don't really see the need of a FrameLayout here, Couldn't we simply use an ImageView that matches its parent and have a scaleType `centerInside` ?

Comment (tbarthel-fr) [inline CameraColorPicker/app/src/kids/res/layout/view_palette_list_page.xml]: sure I totally forgot to remove that. Initially, a additionnal arrow was also contained by this FrameLayout. Gonna removed it right now.

**Comment by tbarthel-fr** [comment]
@vbarthel-fr both useless FrameLayout from color and palette layout has been removed (=

Let you review and merge if it sounds good to you.

**Comment by vbarthel-fr** [comment]
lgtm :+1: 

@tbarthel-fr some conflitcs must be resolved before merging.

**Pull request merged**